viabel/objectives.py

```diff
@@ -178,7 +178,7 @@ def RGE(var_param):
             epsilon_sample = (z_samples - m_mean) / s_scale
 
             # elbo = np.mean(self._model(z_samples) - approx.log_density(var_param, z_samples))
-
+            # TODO: maybe variational_objective function can be called to replace this duplicated part
             if self._use_path_deriv:
```

Comment:
There are a lot of extraneous blank lines. Try no to have blank lines within a function unless it's really necessary for clarity. And use two plan lines between functions and classes